did-aptos/sources/addr_aggregator.move

```diff
@@ -101,27 +101,26 @@ module my_addr::addr_aggregator {
         addr_aggr.description = description;
     }
 
-    // Add addr.
-    public entry fun add_addr(acct: &signer,
-                              addr_type: u64,
-                              addr: String,
-                              pubkey: String,
-                              chains: vector<String>,
-                              description: String) acquires AddrAggregator {
+    public fun add_one_addr(
```

Comment:
typo: rename `add_one_addr` to `do_add_addr`

Reply:
already rename it